L/libcleri/build_tarballs.jl

```diff
@@ -0,0 +1,51 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "libcleri"
+version = v"0.12.1"
+
+# Collection of sources required to complete build
+sources = [
+    GitSource("https://github.com/transceptor-technology/libcleri.git", "a8a1adc9dcf4e889a4d44c17acb20d74d0c6cfe5")
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+export CFLAGS=$(pcre2-config --cflags)
+export LDFLAGS=$(pcre2-config --libs8)
+cd $WORKSPACE/srcdir
+cd libcleri/
+make -C Release -f makefile
+make -C Release -f makefile install INSTALL_PATH=${prefix}
```

Comment:
They build the library with the wrong name for Windows
```suggestion
make -C Release FN="libcleri.${dlext}"
make -C Release FN="libcleri.${dlext}" install INSTALL_PATH=${prefix}
```